assets/script.js

```diff
@@ -0,0 +1,203 @@
+function updatePage(horData, sign) {
+
+
+    sign = sign.toUpperCase();
+
+
+    var $currentHorEl = $("<ul>");
+    $currentHorEl.addClass("curr-hor");
+    $("#horoscope").append($currentHorEl);
+
+    console.log(horData);
+    var color = horData.color;
+    var $currentHorItem = $("<li>");
+    $(".curr-hor").append($currentHorItem);
+    $currentHorItem.append("<h6> Hello " + full_name + "!!. This is your Horoscope for " + today + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Your sun sign is " + sign + " based on your birthday between " + horData.date_range + "</h6>" + "<br>");
+    $currentHorItem.append("<h6>" + horData.description + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Mood: " + horData.mood + " </h6>" + "<br>");
+    $currentHorItem.append("<h6> Your lucky color is: " + horData.color + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Your lucky number is: " + horData.lucky_number + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Your lucky time is: " + horData.lucky_time + "</h6>" + "<br>");
+    console.log("Color is: ", color);
+    if (window.document.title !== "Astrozoders") {
+        tarot();
+    }
+    if (index) {
+
+        historySign = sign;
+        historyName = full_name;
+        console.log("type of historysing: ", typeof (historySign), "type of name: ", typeof (historyName));
+        historySign = JSON.stringify(historySign);
+        localStorage.setItem('sign', historySign);
+        historyName = JSON.stringify(historyName);
+        localStorage.setItem('name', historyName);
+
+    }
+}
+
+//  Function to build the URL for the API call
+function buildURL(sign) {
+
+    console.log("Inside buildURL");
+    if (index) {
+        console.log("index: ", index);
+        full_name = $("#name").val().trim();
+        console.log("Name input: ", full_name);
+        if (full_name === "") {
+            console.log("Name is not entered");
+            $('.modal').modal();
+            $('.modal').modal('open');
+            return;
+        }
+    }
+    clear();
+
+    console.log(sign);
+    var queryURL = "https://aztro.sameerkumar.website?";
+    var queryParams = {};
+    queryParams.sign = sign;
+    queryParams.day = "today";
+    var queryURL = queryURL + $.param(queryParams);
+    console.log("------------------------------------");
+    console.log("QueryURLall", queryURL);
+    console.log("'https://aztro.sameerkumar.website?sign=aries&day=today',");
+    // AjAX request to API to retrieve horoscope info
+    $.ajax({
+        type: 'POST',
+        url: 'https://aztro.sameerkumar.website?sign=aries&day=today',
+        url: queryURL,
+        success: function (horData) {
+            console.log(horData);
+            updatePage(horData, sign);
+        }
+    });
+};
+// this function will be executed when the page is loaded
+$(document).ready(function () {
+
+    $('.sidenav').sidenav();
+    console.log(window.document.title);
+    if (window.document.title === "Tarot") {
```

Comment:
same here, you should have 2 separate files with 2 separate 

```$(document).ready(function()```

statements, but only the relevant code for each page should be included